src/components/searchBox.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component, Fragment } from "react";
+import InputText from "./inputText";
+
+class SearchBox extends Component {
```

Comment:
I think we should use hooks to maintain the state of the components. Because of this we should avoid class components and use functional components instead. You can check https://reactjs.org/docs/hooks-intro.html